cms/templates/js/certificate-details.underscore

```diff
@@ -33,8 +33,10 @@
         <li class="action action-edit">
             <button class="edit"><i class="icon fa fa-pencil" aria-hidden="true"></i> <%= gettext("Edit") %></button>
         </li>
+        <% if (CMS.User.isGlobalStaff) { %>
         <li class="action action-delete wrapper-delete-button" data-tooltip="<%= gettext('Delete') %>">
```

Comment:
As i mentioned purpose of this PR is to hide activate/deactivate and delete buttons it has nothing to with other actions including edit.